cluster/manifests/admission-control-credentials/credentials.yaml

```diff
@@ -4,9 +4,10 @@ metadata:
   name: admission-controller-credentials
   namespace: kube-system
   labels:
-    application: kube-apiserver
+    application: kubernetes
+    component: apiserver
```

Comment:
nit: I think this should be `kube-apiserver` because that is the name of the component in Kubernetes. Similar for `kube-controller-manager` and `kube-scheduler`.

Reply:
Makes sense!